src/ProjectSettings/TagManager.asset

```diff
@@ -3,7 +3,11 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags: []
+  tags:
+  - FxTemporaire
+  - Enemy
```

Comment:
It would be nice if we create an "enum" for tags, this would make it easier to refactor our code when we need to rename a tag.